src/rpc/response.rs

```diff
@@ -426,6 +426,24 @@ impl<'a> HasOwner for Result<CachedResponse, Error<'a>> {
     }
 }
 
+pub(super) fn identity_response<'a, 'b>(req_id: Id<'a>, identity: &'b str) -> HttpResponse {
+    let mut map = HashMap::new();
+    map.insert("identity", identity);
+    let resp = JsonRpcResponse {
+        jsonrpc: "2.0",
+        result: map,
+        id: req_id,
+    };
+
+    let body = serde_json::to_vec(&resp).expect("couldn't serialize identity");
+
+    return HttpResponse::Ok()
+        .append_header(("x-cache-status", "hit"))
+        .append_header(("x-cache-type", "lru"))
```

Comment:
why

Reply:
Because of my inattentiveness :smiling_face_with_tear:, fixed as well